src/conductor/op-conductor/launcher.star

```diff
@@ -25,7 +22,19 @@ def launch(
     el_params,
     cl_params,
     observability_helper,
+    builder_el_params=None,
 ):
+    # Define websocket port for flashblocks support (only if MEV sidecar is present)
+    enable_flashblocks_ws = sidecar_context != None
+    # Allow override via params.websocket_port if provided
+    ws_port = (
+        params.websocket_port
+        if enable_flashblocks_ws
+        and hasattr(params, "websocket_port")
```

Comment:
Couple of things here:

- it seems very implicit this logic - in general the input parser should be handling all the defaults like e.g. the `8546` value here
- there is a convention for ports - they live in `params.ports` - and again if the port number is something that can be modified by the user, input parser should take care of sanitizing it and putting it into the `params.ports` dict
- `hasattr` need not be used - input parsers ensure that all the properties are present. If a property is not present, the script should fail - otherwise we are potentially obscuring script errors